fortitude/src/printer.rs

```diff
@@ -78,66 +155,171 @@ impl Printer {
     pub(crate) fn write_once(
         &self,
         num_files: usize,
-        diagnostics: &[DiagnosticMessage],
+        diagnostics: &Diagnostics,
         writer: &mut dyn Write,
     ) -> Result<()> {
         // TODO: implement tracking of fixables
```

Comment:
I think this TODO has been resolved

Reply:
The suggestion is pointing at the wrong lines. It was meant to be the TODO at the start of this function.